tests/run_test.go

```diff
@@ -28,7 +28,7 @@ func runOneToOneTestWithAuth(tb testing.TB, options *serverOptions, tests func(*
 	})
 }
 
-// runOneToOneTestWithData runs a test with one account and one connection. A mailbox is created with test data.
+// runOneToOneTestWithData runs a test with one account and one connection. Apply mailbox is created with test data.
```

Comment:
Wow yes i see more `A` -> `Apply`, definitely not expected :D

Reply:
Fixed.